Orange/widgets/regression/owsgdregression.py

```diff
@@ -197,6 +197,16 @@ def _on_lrate_changed(self):
         for spin, enabled in zip(self._lrate_params, mask):
             spin.setEnabled(enabled)
 
+    def send_report(self):
+        """ report learner name,model and data"""
+        self.report_items((("Name",self.learner_name),))
+        self.report_items("Model Parameters",(("Loss function to be used ",self.loss_function)
+                                            ,("Penalty ",self.penalty_type)
+                                            ,("Learning rate",self.learning_rate)
+                                            ,("Constant",self.constant))
```

Comment:
Please make this PEP-8 conformant (space after comma) and indented correctly (by one more space character). It's important to strive to leave code in better shape than you get it, not worse. :smiley: